DATA/SHIPS/loadouts.ini

```diff
@@ -1524,37 +1524,34 @@ equip = DockingLightRedSmall, HpDockLight02
 
 [Loadout]
 nickname = MSN10_Fischer
-archetype = rh_freighter
-equip = ge_rfr_engine_01
-equip = npc_shield02_mark02, HpShield01
+archetype = rh_fighter
+equip = ge_rf_engine_01
+equip = npc_shield02_mark05, HpShield01
 equip = infinite_power
 equip = ge_s_scanner_02
 equip = ge_s_tractor_01
 equip = ge_s_thruster_01, HpThruster01
-equip = armor_scale_2
-equip = co_gun01_mark02, HpWeapon01
-equip = co_gun01_mark02, HpWeapon02
-equip = co_turret01_mark02, HpTurret01
-equip = co_turret01_mark02, HpTurret02
-equip = co_turret01_mark02, HpTurret03
-equip = co_turret01_mark02, HpTurret04
-equip = mine01_mark01, HpMine01
-cargo = mine01_mark01_ammo, 20
+equip = rh_gun01_mark03, HpWeapon01
+equip = rh_gun01_mark03, HpWeapon02
+equip = rh_gun01_mark03, HpWeapon03
+equip = missile01_mark04, HpWeapon04
+cargo = missile01_mark04_ammo, 20
+equip = mine01_mark04, HpMine01
+cargo = mine01_mark04_ammo, 10
 equip = ge_s_cm_01, HpCM01
 cargo = ge_s_cm_01_ammo, 20
+equip = cloak_fighter, HpCloak01
 equip = LargeWhiteSpecial, HpHeadlight
 equip = SlowSmallYellow, HpRunningLight01
 equip = SlowSmallYellow, HpRunningLight02
 equip = SlowSmallYellow, HpRunningLight03
 equip = SlowSmallYellow, HpRunningLight04
 equip = SlowSmallYellow, HpRunningLight05
 equip = SlowSmallYellow, HpRunningLight06
-equip = SlowSmallYellow, HpRunningLight07
 equip = contrail01, HpContrail01
 equip = contrail01, HpContrail02
 equip = DockingLightRedSmall, HpDockLight01
 equip = DockingLightRedSmall, HpDockLight02
-cargo = commodity_construction_machinery, 100, hpcargo01
```

Comment:
Please undo all the `MSN10_Fischer` changes. This has previously been updated to make the loadout consistent with the mission 10 voice lines.